File: Java/Array/Count Inversions.java
//https://www.geeksforgeeks.org/problems/inversion-of-array-1587115620/1
int n=arr.size(), result=0,i,j;
        for(i=0;i<n-1;i++){
            for(j=i+1;j<n;j++){
                if(arr[i]>arr[j]){
                    result++;
                }
            }
        }
        return result;

File: Python/Array/Count Inversions.py
//https://www.geeksforgeeks.org/problems/inversion-of-array-1587115620/1
class Solution:
    # Function to count inversions in the array.
    def inversionCount(self, arr):
        # Your Code Here
        n = len(arr)
        result = 0
        for i in range(n - 1):
            for j in range(i + 1, n):
                if arr[i] > arr[j]:
                    result += 1
        return result
